.buildkite/scripts/shared.ps1

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env powershell
+
+#Requires -Version 5
+
+class HabShared {
```

Comment:
The powershell idiomatic way of sharing functions is to use a module. You don't yet see classes used much in powershell but their value is really in expressing a domain object model that uses some state. You could easily make this a module by declaring the functions with the `function` keyword and saving this file with a `.psm1` extensions. Then the files that use this would change `. "$PSScriptRoot\shared.ps1"` to `Import-Module "$PSScriptRoot\shared.ps1"` and then its functions come into scope.